terraform/gcp/modules/mysql/mysql.tf

```diff
@@ -261,12 +261,12 @@ resource "google_sql_user" "breakglass_iam_group" {
   type     = "CLOUD_IAM_GROUP"
 }
 
-resource "google_project_iam_binding" "breakglass_iam_group_permissions" {
```

Comment:
Iam binding was authoritative, was there anything wiped out?

Reply:
because `mysql-shard` reuses the same `mysql` code, it isn't happy with having two authoritative clauses in the same plan.